frontend/public/load-test.sw.js

```diff
@@ -8,22 +8,62 @@ const uuidFor = () => {
   });
 };
 
+const pageFromToken = (continueToken) => continueToken ? parseInt(continueToken.replace('toPage', ''), 10) : 0;
+
 const clonePod = (pod, num) => ({...pod, metadata: {...pod.metadata, name: `${pod.metadata.name}-clone-${num}`, uid: uuidFor()}});
 
-let multiplicationFactor = 50;
+const strippedURLFor = (request) => {
+  const url = new URL(request.url);
+  url.searchParams.delete('limit');
+  url.searchParams.delete('continue');
+
+  return url.toString();
+};
+
+let multiplicationFactor = 20;
+
+/**
+ * Only match core resource list requests.
+ */
+const workloads = new Set()
+  .add(/\/api\/kubernetes\/apis\/apps\/v1(\/namespaces\/.*)?\/daemonsets(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/apis\/apps\/v1(\/namespaces\/.*)?\/deployments(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/apis\/apps\/v1(\/namespaces\/.*)?\/replicasets(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/api\/v1(\/namespaces\/.*)?\/replicationcontrollers(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/api\/v1(\/namespaces\/.*)?\/persistentvolumeclaims(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/apis\/batch\/v1(\/namespaces\/.*)?\/jobs(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/apis\/batch\/v1beta1(\/namespaces\/.*)?\/cronjobs(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/api\/v1(\/namespaces\/.*)?\/pods(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/api\/v1(\/namespaces\/.*)?\/configmaps(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/api\/v1(\/namespaces\/.*)?\/secrets(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/api\/v1(\/namespaces\/.*)?\/resourcequotas(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/apis\/extensions\/v1beta1(\/namespaces\/.*)?\/ingresses(\?(.*))?$/)
+  .add(/\/api\/kubernetes\/api\/v1(\/namespaces\/.*)?\/services(\?(.*))?$/);
```

Comment:
This only works for <code>/pods</code> list currently, right?

Reply:
It did inititally, now it works for all of the resources in this set (the most common upstream k8s list views). That way we have more flexibility when developing virtual scrolling and such.

Reply:
Asking because you still have pod specific logic in here https://github.com/openshift/console/pull/58/files#diff-bf66568db68490cee5e2acdafdf48110R57

Reply:
Ah, the function is called `clonePod`, but it isn't specific to pods because it just changes the `metadata.name` and `metadata.namespace` fields. I renamed it to be less confusing, but it works regardless if you look at the "Deployments" view or any other.